### PR TITLE
dotCMS/core#16456 Render form to edit page with the dotcms library

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.tsx [hunk 1]
```diff
@@ -93,25 +94,25 @@ export class DotKeyValueComponent {
...
                     class={getErrorClass(this.status.dotValid)}
                     disabled={this.isDisabled()}
-                    id={this.name}
-                    name="key"
+                    id={getId(this.name)}
+                    name='key'
                     onInput={(event: Event) => this.setValue(event)}
                     placeholder={this.keyPlaceholder}
-                    type="text"
+                    type='text'
```
Comment (hmoreras) [inline projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.tsx]: keep double quote ?

File: projects/dotcms-field-elements/src/components/dot-key-value/key-value-table/key-value-table.e2e.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ describe('key-value-table', () => {
 
     it('renders', async () => {
         // tslint:disable-next-line:max-line-length
-        const expectedMarkup = `<key-value-table class=\"hydrated\"><table><tbody><tr><td><button type=\"button\" id=\"undefined_undefined_0\" class=\"dot-key-value__delete__button\"><div class=\"dot-field__label\"><label for=\"undefined_undefined_0\">Delete</label></div></button></td><td></td><td></td></tr></tbody></table></key-value-table>`;
+        const expectedMarkup = `<key-value-table class=\"hydrated\"><table><tbody><tr><td><button type=\"button\" id=\"undefined_undefined_0\" class=\"dot-key-value__delete__button\"><div class=\"dot-field__label\"><label for=\"dot-undefined_undefined_0\">Delete</label></div></button></td><td></td><td></td></tr></tbody></table></key-value-table>`;
```
Comment (hmoreras) [inline projects/dotcms-field-elements/src/components/dot-key-value/key-value-table/key-value-table.e2e.ts]: `id=\"undefined_undefined_0\"` ? same for label.

File: projects/dotcms-field-elements/src/components/dot-select/dot-select.e2e.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { newE2EPage, E2EElement, E2EPage } from '@stencil/core/testing';
 import { EventSpy } from '@stencil/core/dist/declarations';
 
-describe('dot-select', () => {
+fdescribe('dot-select', () => {
```
Comment (hmoreras) [inline projects/dotcms-field-elements/src/components/dot-select/dot-select.e2e.ts]: remove.

File: projects/dotcms-field-elements/src/components/dot-textfield/dot-textfield.tsx [hunk 4]
```diff
@@ -3,11 +3,12 @@ import Fragment from 'stencil-fragment';
 import { DotFieldStatus, DotFieldValueEvent, DotFieldStatusEvent, DotLabel } from '../../models';
```
Comment (hmoreras) [inline projects/dotcms-field-elements/src/components/dot-textfield/dot-textfield.tsx]: lets add the `type` prop in this commit to support email ?

File: projects/dotcms-field-elements/src/components/dot-key-value/key-value-table/key-value-table.e2e.ts (hunk 2 above)
Comment (fmontes) [reply projects/dotcms-field-elements/src/components/dot-key-value/key-value-table/key-value-table.e2e.ts]: Master have that, @alfredo-dotcms? https://github.com/dotCMS/core-web/blob/master/projects/dotcms-field-elements/src/components/dot-key-value/key-value-table/key-value-table.e2e.ts#L19

Comment (alfredo-dotcms) [reply projects/dotcms-field-elements/src/components/dot-key-value/key-value-table/key-value-table.e2e.ts]: yes, this needs work, I'll fix it

**Pull request merged**